### PR TITLE
ADD nodeLabel prop to checkboxField

### PR DESCRIPTION
**Pull request by andresin87**
**Resolves** [https://jira.scmspain.com/browse/SUIC-451](https://jira.scmspain.com/browse/SUIC-451)
**Distro** [http://localhost:5000/workbench/molecule/checkboxField/demo](http://localhost:5000/workbench/molecule/checkboxField/demo)

This feature gives the posibility to set HTMLelements to the label props.

There are 2 different components modiffied in this feature which do not give us the possibility of checking the visible expected result. checkboxField needs a valid published field version which doesn't exist yet until itsown is already merged. 🤯

Works in local
image

### CODE REVIEW

**Review by carlosvillu — APPROVED** [review]
Good work !!

File: components/molecule/field/src/index.js [hunk 1]
```diff
@@ -12,16 +12,41 @@ const BASE_CLASS = 'sui-MoleculeField'
...
 const CLASS_LABEL_CONTAINER = `${BASE_CLASS}-labelContainer`
 
+const MoleculeLabel = ({
+  label,
+  nodeLabel,
+  typeValidationLabel,
+  name,
+  onClickLabel
+}) => {
+  let response = null
```
Comment (davecarter) [inline components/molecule/field/src/index.js:26]: You can use a short-circuit evaluation here so there's no need to mutate `response`:

```javascript
  const MoleculeLabel = ({
    label,
    nodeLabel,
    typeValidationLabel,
    name,
    onClickLabel
  }) =>
    label &&
    ((
      <AtomLabel
        type={typeValidationLabel}
        name={name}
        text={label}
        onClick={onClickLabel}
      />
    ) ||
      (nodeLabel && (
        <div className={CLASS_NODE_LABEL_CONTAINER}>{nodeLabel}</div>
      )))
```

File: test/molecule/checkboxField/index.js [hunk 2]
```diff
@@ -7,16 +7,93 @@
...
+
+const setup = setupBuilder(MoleculeCheckBoxField)
+
 describe('molecule/checkboxField', () => {
-  it('Render', () => {
-    // Example TO BE DELETED!!!!
-    // const {getByRole} = render(<AtomButton>HOLA</AtomButton>)
-    // expect(getByRole('button')).to.have.text('HOLA')
-    expect(true).to.be.eql(false)
+  describe('props', () => {
```
Comment (davecarter) [inline test/molecule/checkboxField/index.js:29]: Awwesome testing 👏 👏

**Review by davecarter — APPROVED** [review]

**Review by jcger — APPROVED** [review]

File: test/molecule/checkboxField/index.js [hunk 3]
```diff
@@ -7,16 +7,93 @@
 // import React from 'react'
```
[the remark this answers is not in the document]
Comment (andresin87) [reply test/molecule/checkboxField/index.js]: DONE

File: test/molecule/checkboxField/index.js (hunk 2 above)
Comment (andresin87) [reply test/molecule/checkboxField/index.js:29]: 😉

File: components/molecule/field/src/index.js (hunk 1 above)
Comment (andresin87) [reply components/molecule/field/src/index.js:26]: I made it as is dues to readability reasons

**Pull request merged**